app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,7 @@
 # : true
 
+# : true
```

Comment:
- Hello, Can you remove this comment code from all files in controllers? I am requesting this because it is creating confusion. Also, it is making your controller's codes unclean.